### PR TITLE
Arranging all sections in alphabetical order  `enhancement`

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
arranging all sections one by one in alphabet order

### CODE REVIEW

**Review by Ileriayo — CHANGES_REQUESTED** [review]
I think this is a really valuable PR.
Thanks for taking the time to contribute!

Kindly resolve the merge conflict, so that your PR can be merged.
Thanks again.

**Comment by gurjeetsinghvirdee** [comment]
> I think this is a really valuable PR. Thanks for taking the time to contribute!
> 
> Kindly resolve the merge conflict, so that your PR can be merged. Thanks again.

My pleasure, & I want to collaborate with this repo

File: README.md [hunk 1]
```diff
@@ -710,27 +1202,21 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
...
+| Indeed       | ![Indeed](https://img.shields.io/badge/indeed-003A9B?style=for-the-badge&logo=indeed&logoColor=white)         | `![Indeed](https://img.shields.io/badge/indeed-003A9B?style=for-the-badge&logo=indeed&logoColor=white)`              |
+| Upwork       | ![Upwork](https://img.shields.io/badge/UpWork-6FDA44?style=for-the-badge&logo=Upwork&logoColor=white)                      | `![Upwork](https://img.shields.io/badge/UpWork-6FDA44?style=for-the-badge&logo=Upwork&logoColor=white) `                     |
 
 [![](https://img.shields.io/badge/back%20to%20top-%E2%86%A9-blue)](#shortcuts)
 
-### Smartphone Brands
 
 | Name       | Badge                                                                                                                 | Markdown                                                                                                                |
 | ---------- | --------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
```
Comment (Ileriayo) [inline README.md]: Some sections like this one have been left orphaned.

**Review by Ileriayo — CHANGES_REQUESTED** [review]
Hey, great work.
However, some badges have been orphaned.
Also, some sections are not arranged alphabetically (see sample screenshot below)
Screenshot 2022-01-11 at 20 58 31

**Review by Ileriayo — APPROVED** [review]
LGTM - Looks good to me.
Thanks for contributing!

**Pull request merged**